src/ui/components/atoms/label-variables/Variables.tsx

```diff
@@ -0,0 +1,33 @@
+import './styles.scss';
+import type { IProps } from './types/IProps';
+
+function Variables({ text = 'ARTICULOS', colorVariant = 'primary', href }: IProps) {
+  const getColorClassName = () => {
+    switch (colorVariant) {
+      case 'main':
+        return 'variables--main';
+      case 'secondary':
+        return 'variables--secondary';
+      default:
+        return 'variables--primary';
+    }
+  };
```

Comment:
I don't see it necessary to create a function to translate the data there are too many different approaches to this more easily.
something like this.
```
const colorClassNames: Record<string, string> = {
  main: 'variables--main',
  secondary: 'variables--secondary',
  default: 'variables--primary'
};
```
and to use 
const className = colorClassNames[colorVariant]